### PR TITLE
var -> let

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md?ref=441126bbb0abfcb18fe18b44e13bba1f68fa34af)

File: javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md [hunk 1]
```diff
@@ -45,17 +39,17 @@ Of course you only need to specify those you want to intercept otherwise the def
 Here is an example of intercepting delete calls:
```
Comment (Stefan-Stojanovic) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: line 37 ( cant add a comment on it) "Of course" -> "Of course,"

File: javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md [hunk 2]
```diff
@@ -14,8 +10,6 @@ aspects:
   - new
   - workout
 
-inAlgoPool: false
-
 links:
   - '[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy){website}'
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]:
```suggestion
  - '[Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy){documentation}'
```

File: javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md [hunk 3]
```diff
@@ -85,8 +79,7 @@ Then we can deny access to the proxy by calling the revoke method:
 ```javascript
 p.revoke();
 p.a;
-// VM181:1 Uncaught TypeError:
-// p.revoke is not a function(...)
+// TypeError: p.revoke is not a function
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md]: image

updated the error message

**Pull request merged**